examples/basecoin/app/init_baseapp.go

```diff
@@ -26,3 +33,68 @@ func (app *BasecoinApp) initBaseAppTxDecoder() {
 		return tx, nil
 	})
 }
+
+// We use GenesisAccount instead of types.AppAccount for cleaner json input of PubKey
```

Comment:
is this because we don't have go-wire JSON yet ?

Reply:
no, json representation of the new PubKey is a byte array... I'd obviously rather not use an input struct, if you have any other ideas lmn

Reply:
Oh, we don't even need PubKeys for accounts in the genesis file!